src/cljs/athens/electron.cljs

```diff
@@ -325,6 +325,46 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+  ;; Update index.transit    
+  (defn write-db-index [file filepath]
+    (.. fs (writeFile filepath file (fn [e] (dispatch [:boot/desktop]))))
+  )
+  
+  ;; Create new index.transit db
+  (defn create-db-index [file filepath]
+    (.. fs (writeFile filepath file (fn [e] (dispatch [:fs/open-dialog]))))
+  ) 
+  
+  (defn open-dialog-index
+    "Allow user to open a Backup file."
+    [filepath]
+    (js/alert "Your index.transit is corrupted, please open up a backup")
+    (let [res  (.showOpenDialogSync dialog (clj->js {:properties ["openFile"]
+                                                     :filters    [{:name "Transit" :extensions ["bkp"]}]}))
```

Comment:
`.transit.bkp`?

Reply:
@phrohdoh Hi yes this allowed you to filter and select only the backup files 
as for the extension is bkp and the target name is : bkp-filename (str time "-" (os-username) "-" "index.transit.bkp")
hope i answer your request

Reply:
@phrohdoh do not use this one since there was some updates in between i had some messages. 
use instead #2 above : https://github.com/athensresearch/athens/pull/1230 
thanks